src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -524,7 +529,7 @@ internal void MoveChildTo(int oldIndex, int newIndex)
 		private void OnAddingChild(UIElement child)
 		{
 			if (FeatureConfiguration.FrameworkElement.WasmUseManagedLoadedUnloaded
-				&& IsLoaded
+				&& IsLoaded || IsLoading
```

Comment:
This seems risky, precedence-wise. Maybe parentheses would be welcome.

Reply:
Indeed, fixed